CHANGELOG.md

```diff
@@ -6,17 +6,56 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [3.0.0] - 2022-03-30
+Amazon Chime SDK JS 3.0.0 is here !! 🎉🎉🎉
```

Comment:
`Chime SDK JS 3.0.0` => `Chime SDK for JavaScript v3`

(We used `v2` in the [v2 migration guide](https://aws.github.io/amazon-chime-sdk-js/modules/migrationto_2_0.html). An alternative option is just `3`, `3.0.0`, `version 3`.)